docs/about/index.rst

```diff
@@ -1,32 +1,197 @@
 .. _about:
 
-About
-=====
+About Fairlearn
+===============
+
+Fairlearn is a project of the
+`Fairlearn Organization <https://github.com/fairlearn/governance/blob/main/ORG-GOVERNANCE.md>`_
+and follows the
+`Fairlearn Organization's Project Governance <https://github.com/fairlearn/governance/blob/main/PROJECT-GOVERNANCE.md>`_.
```

Comment:
I'm not sure I like this as a first sentence on the about page - referring to governance structures doesn't really sound very welcoming to me.

Reply:
I see your point. Would you be okay if we just dropped `and follows ...`? I think we need to mention somewhere that this project is part of the Fairlearn Org.

If you'd rather we don't mention Fairlearn Org at this place, we can just move this entire sentence to "Governance" subsection (right now it just repeats the second part of the sentence).

Reply:
Moving the sentence to "Governance" sounds perfect!